dialogy/plugins/text/list_search_plugin/__init__.py

```diff
@@ -204,8 +204,8 @@ def search_regex(
                     final_match = match_value
                     match_span = result.span()
         if final_match:
-            return (match_text, entity_type, final_match, match_span, float(0))
-        return ("", entity_type, "", (0, 0), float(0))
+            return (match_text, entity_type, final_match, match_span, 1.0)
```

Comment:
is this change intentional? float(0) -> 1.0?

Reply:
yes, idea is that since the regex match is 100% match we should keep the score as 1.0, earlier it was 0. but we were not removing the matches with score 0 earlier now we are hence this is something I did intentionally